raster/r.regression.line/main.c

```diff
@@ -154,6 +198,32 @@ int main(int argc, char *argv[])
         fprintf(stdout, "   sdX (Standard deviation of map1): %f\n", sdX);
         fprintf(stdout, "   meanY (Mean of map2): %f\n", meanY);
         fprintf(stdout, "   sdY (Standard deviation of map2): %f\n", sdY);
+        break;
+
+    case JSON:
+        G_json_object_set_number(root_object, "a", A);
+        G_json_object_set_number(root_object, "b", B);
+        G_json_object_set_number(root_object, "R", R);
+        G_json_object_set_number(root_object, "N", count);
+        G_json_object_set_number(root_object, "F", F);
+        G_json_object_set_number(root_object, "x_mean", meanX);
+        G_json_object_set_number(root_object, "x_stddev", sdX);
+        G_json_object_set_number(root_object, "y_mean", meanY);
+        G_json_object_set_number(root_object, "y_stddev", sdY);
```

Comment:
I've recently noticed a problem with r.univar that applies here as well - if your input raster is null or you are running this in a region that doesn't match the input rasters, the values are nan and it silently fails for the JSON, resulting in:
```
{
    "N": 0
}
```
while shell gives you:
```
a=-nan
b=-nan
R=-nan
N=0
F=-nan
meanX=-nan
sdX=-nan
meanY=-nan
sdY=-nan
```

Missing keys are unexpected, so we need to explicitly test for these cases. I think we already ran into this either in g.proj or r.proj.